NuRadioReco/detector/ARIANNA/arianna_detector_200.json

```diff
@@ -0,0 +1,279 @@
+{
+    "_default": {},
+    "channels": {
+        "1": {
+            "adc_id": null,
+            "adc_n_samples": 256,
+            "adc_nbits": null,
+            "adc_sampling_frequency": 1.0,
+            "adc_time_delay": null,
+            "amp_type": "300",
+            "ant_comment": "close to center, pointing downwards (1/4)",
+            "ant_deployment_time": "{TinyDate}:2018-12-05T00:00:00",
+            "ant_orientation_phi": 0.0,
+            "ant_orientation_theta": 180.0,
+            "ant_position_x": 2.1,
+            "ant_position_y": 2.1,
+            "ant_position_z": -3,
+            "ant_rotation_phi": 45.0,
+            "ant_rotation_theta": -90.0,
+            "ant_type": "createLPDA_100MHz",
+            "cab_id": "17-09",
+            "cab_length": 5.0,
+            "cab_reference_measurement": null,
+            "cab_time_delay": 19.8,
+            "cab_type": "LMR_400",
+            "channel_id": 0,
+            "commission_time": "{TinyDate}:2018-12-05T00:00:00",
+            "decommission_time": "{TinyDate}:2038-01-01T00:00:00",
+            "station_id": 101
+        },
+        "2": {
+            "adc_id": null,
+            "adc_n_samples": 256,
+            "adc_nbits": null,
+            "adc_sampling_frequency": 1.0,
+            "adc_time_delay": null,
+            "amp_reference_measurement": "300-03-01",
```

Comment:
can you remove the amp reference measurement fields? This is a future station where we don't know the amps yet

Reply:
okay, i set them to "null"